lib/qx/tool/cli/commands/contrib/Update.js

```diff
@@ -203,7 +202,7 @@ qx.Class.define("qx.tool.cli.commands.contrib.Update", {
               let data = qooxdoo_data.data;
               if (typeof data == "string") {
                 try {
-                  data = jsonlint.parse(data);
+                  data = JSON.parse(data);
```

Comment:
also requires `qx.tool.compiler.utils.Json.parseJson` because `qooxdoo.json` is hand edited